### DISCUSSION TITLE
Pass "data" attribute from GraphNode to renderNode callback.

### DISCUSSION THREAD

**Discussion by popovitsj** [reaviz/reagraph#119] [Ideas]
Unless I'm missing something, there doesn't seem to be a way to access the "data" on a GraphNode in the renderNode callback.

It would be helpful to add it here, to make it easier to render custom nodes.

**Comment by amcdnl** [comment]
I thought it did - I will double check.

**Reply by popovitsj** [reply]
I guess the issue is that `InternalGraphNode` is included in the `NodeRendererProps`, not `GraphNode`.

Additionally, I noticed that `GraphNode` is not a generic, but `GraphElementBaseAttributes` is. It seem to me we should pass the generic from Graphnode, like this here: https://github.com/reaviz/reagraph/blob/ddd08b8e8acc365dd562ff22d36f889621c4ee67/src/types.ts#L31

```
export interface GraphNode<T = any> extends GraphElementBaseAttributes<T> {
```
